index.md

```diff
@@ -45,3 +45,4 @@ possible by generous contributions of
 ## GitHub Action Contributors
 <ul class="list-style-none">  <li class="d-inline-block mr-1"> <a href="https://github.com/arsulegai"><img src="https://avatars.githubusercontent.com/u/27664223?v=4" width="32" height="32" alt="27664223"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/hyperledger-bot"><img src="https://avatars.githubusercontent.com/u/76175814?v=4" width="32" height="32" alt="76175814"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/nidhi-singh02"><img src="https://avatars.githubusercontent.com/u/38173192?v=4" width="32" height="32" alt="38173192"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/ryjones"><img src="https://avatars.githubusercontent.com/u/466142?v=4" width="32" height="32" alt="466142"></a> </li>  </ul>
 
+## [Community statistics](https://insights.lfx.linuxfoundation.org/projects/hyperledger-f) from Hyperledger 
```

Comment:
1. @Swastyy this link did not take me to Hyperledger Foundation's statistics. Could you please check again?
What worked for me is https://insights.lfx.linuxfoundation.org/projects/hyperledger-f/dashboard

2. It would be nice to have a nice title and then the contents can have the text with a link to lfx insights.